### PR TITLE
Add pageres spec

### PR DESCRIPTION
**Pull request by Monkeyank**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature
**What is the new behavior (if this is a feature change)?**
Adds `pageres` spec

### CODE REVIEW

File: dev/pageres.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  name: "pageres",
+  description: "Capture website screenshots",
+  options: [
+    {
+      name: ["--verbose", "-v"],
+      description: "Verbose output to see errors if you need to troubleshoot",
+    },
+    {
+      name: ["--crop", "-c"],
+      description: "Crop to the set height",
```
Comment (QuiiBz) [inline dev/pageres.ts:11]: Does this option take an argument?

File: dev/pageres.ts [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    {
+      name: "--hide",
+      description: "Hide DOM element. Can be set multiple times",
+      args: {
+        name: "element",
+      },
+    },
+    {
+      name: "--no-crop",
+      description: "Override a global crop option within a group",
```
Comment (QuiiBz) [inline dev/pageres.ts:49]: This option can exclude the `--crop` one

File: dev/pageres.ts [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    },
+    {
+      name: "--css",
+      description:
+        "Apply custom CSS to the webpage. Specify some CSS or the path to a CSS file",
+      args: {
+        name: "string",
```
Comment (QuiiBz) [inline dev/pageres.ts:56]: Looks like we can have a template of `filepaths` here

**Review by QuiiBz — CHANGES_REQUESTED** [review]

File: dev/pageres.ts (hunk 1 above)
Comment (Monkeyank) [reply dev/pageres.ts:11]: This doesn't take an argument

**Review by QuiiBz — APPROVED** [review]
LGTM!

**Pull request merged**